subcommittees/mentoring/README.md

```diff
@@ -1,73 +1,100 @@
 # The Mentoring Subcommittee
 
 ## Purpose and Scope
+
 The Mentoring Subcommittee was established in 2015 to develop and maintain a mentorship program to support instructors as they progress through training, teaching, curriculum development, and other community-related activities. We help promote community-building and networking by providing (virtual) spaces where instructors from all over the world can share teaching success stories and discuss strategies for overcoming challenges. 
 
-## What We Do 
+## What We Do
 
 ### Instructor Discussion Sessions
-Each week, we host four instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
+
+Each week, volunteers host two instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
 
 To find an upcoming session, visit this link: http://pad.software-carpentry.org/instructor-discussion
 
-### Instructor-Helper Retreat
-In November 2015, Software Carpentry and Data Carpentry instructors and helpers got together together at sites around the world for a day of sharing skills, trying out new lessons and ideas and discussing all things instructor and helper related. 
+### Mentoring Program
 
-To see the schedule for the 2015 retreat, visit: http://swcarpentry.github.io/instructor-retreat-2015/
+This program supports instructors who are new to our community by matching them with a personal Mentor. Mentors will guide small groups (~4-5) of Mentees, holding meetings for their Mentees to do things like practice teaching, ask questions about workshop logistics, and brush up on lesson content, depending on their needs and interests. 
 
-### Archived: Pre- and Post- Workshop Debriefing Session
+See [this page](https://github.com/datacarpentry/mentoring-program/blob/master/program-outline.md) for a full outline of the program.  
 
-Before launching the Instructor Discussion Sessions, we used to host separate [pre- and post-workshop debriefing sessions](http://software-carpentry.org/blog/categories/#debriefing). We are grateful to all the hosts and participants who attended these sessions.
+### Mentoring Committee Meetings
 
-## Mailing List
+The mentoring committee itself meets regularly to evaluate current activities, and propose
+new activities.  Minutes of past meetings are available [here][minutes].  All community 
```

Comment:
The template's [`[here](minutes)`][1] [works][2] (with parens), and then you don't have to specify the full path [down here][3].

[1]: https://github.com/swcarpentry/board/blame/09b7e2264b85510cfeda5653e2a8611893614df1/subcommittees/template/README.md#L32
[2]: https://github.com/swcarpentry/board/blob/09b7e2264b85510cfeda5653e2a8611893614df1/subcommittees/template/README.md#meetings
[3]: https://github.com/swcarpentry/board/pull/175/files#diff-f161ee6b139bf0d7d8e110ccfb676c1bR100